reference/wire-protocol.md

```diff
@@ -17,11 +17,32 @@ The message body is a sequence of fields as described by the associated schema.
 
 ## Setup
 
-As soon as a connection is established, the client must send to the server a single word containing the protocol version it wishes to use.
+As soon as a connection is established, the client must send to the server a single word containing the protocol version it wishes to use. The current protocol version is 1 (in little-endian representation).
 
 ## Conversation
 
-After the setup, communication between client and server happens by message exchange. Typically the client will send to the server a message containing a request and the server will send to the client a message containing a response.
+After the setup, communication between client and server happens by message exchange. Typically the client will send to the server a message containing a request and the server will send to the client a message containing a response. Any request may result in an failure response (type `0`); otherwise, the correspondence between request and response types is as follows:
```

Comment:
```suggestion
After the setup, communication between client and server happens by message exchange. Typically, the client will send to the server a message containing a request and the server will send to the client a message containing a response. Any request may result in a failure response (type `0`); otherwise, the correspondence between request and response types is as follows:
```